docs/solana/options.rst

```diff
@@ -1,5 +1,202 @@
+- `executables`: List of compiled binary files, which are the target of the Rust program.
+- `success`: Boolean flag indicating if the build phase passed successfully.
+
+**When to use it?**
+
+Use this mode to prove properties on source code while providing an automatic compilation method. This is especially useful during development when files are modified frequently.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script>
+
+precompiled binary
+~~~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Runs formal verification of specified properties on a precompiled Rust project by providing a path to the binary target file.
+
+**When to use it?**
+
+Use this mode to prove properties on source code without recompiling the project for every execution. Ideal when files are stable and unchanged.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver <path_to_binary_file>
+
+Most Frequently Used Options
+----------------------------
+.. _solana_inlining:
+--solana_inlining
+~~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Provides the prover with a list of paths to inlining files for Solana contracts. These files are parsed and used to prove properties.
+
+**When to use it?**
+
+TODO: @Jorge please advise.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --solana_inlining <path_to_inlining_file>
+
+.. _solana_summaries:
+--solana_summaries
+~~~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Provides the prover with a list of paths to summary files for Solana contracts. These files are parsed and used to prove properties.
+
+**When to use it?**
+
+TODO: @Jorge please advise.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --solana_summaries <path_to_summaries_file>
+
+.. _cargo_features:
+--cargo_features
+~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Provides the prover with a whitespace-separated list of extra `Cargo features <https://doc.rust-lang.org/cargo/reference/features.html>` passed to the build script.
+These features are then passed to ``cargo`` to build the project.
+
+**When to use it?**
+
+Use it when there is a need to enable a specific `Cargo feature <https://doc.rust-lang.org/cargo/reference/features.html>` to compile the source code.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --cargo_features <feature_1> <feature_2>
+
+.. _msg:
+--msg
+~~~~~
+
+**What does it do?**
+
+Adds a description message to your run, similar to a commit message. This message appears on the Prover dashboard and in the rule report.
+Note that you need to wrap your message in quotes if it contains spaces.
```

Comment:
I find the "note" confusing. `msg` takes one argument. If I want a string with spaces, I will pass it based on the shell that I'm using. Most often it is achieved by putting the string in quotes. Since the documentation is written for a command line user, they know what to do.